packages/graph-editor/src/components/flow/wrapper/nodeV2.tsx

```diff
@@ -178,6 +178,7 @@ export const WrapNode = (
             mappedInput,
             state,
             value,
+            ephemeralState,
```

Comment:
I got the type error here that the function expects 4 arguments. We are calling `(nodeDef.mapOutput || defaultMapOutput)` where the first one has 4 required parameters and the second one has 3 so technically we *should* provide the 4th one which is `ephemeralState`.

It is still a bit weird because I believe I haven't changed any code which affects the type check here. Please correct me if I'm wrong.